addons/hr_expense/views/hr_expense_views.xml

```diff
@@ -537,7 +538,8 @@
                             </group>
                             <group string="Accounting">
                                 <field name="property_account_expense_id" groups="account.group_account_readonly"/>
-                                <field name="supplier_taxes_id" domain="[('price_include', '=', True)]" widget="many2many_tags" context="{'default_type_tax_use':'purchase'}"/>
+                                <field name="supplier_taxes_id" domain="[('price_include', '=', True)]" widget="many2many_tags" context="{'default_type_tax_use':'purchase'}" 
```

Comment:
Just noticed that here something is missing from the context :see_no_evil: 
 Put here as well `'default_price_include': 1` in the context. It is needed, because all the calculations in expense implies that taxes are included in the price.